sources/3.0.0/sections/12-metadata.adoc

```diff
@@ -401,7 +401,7 @@ _False_ indicates the dataset *is* intended to be used for navigation.
 
 |Attribute
 |boundingBox
-|The extent of the datast limits
+|The extent of the dataset limits defined as a rectangle coincident with the outermost cell boundaries of the dataset.
```

Comment:
How many data set limits are there?
One, right? 
Shouldn't it be "limit" instead of "limits"?

Reply:
Can an English speaking PT member double check this for us (@hasel001). This one is connected to section 4, above.

Reply:
I agree with "limits" (as above).

I can certainly see it from both perspectives. Although I don't think we are limited to 1 reasonable answer, we must certainly be internally consistent. Looking back as far as Ed. 2.1.0, it appears we have used "limit" to refer to scalar values (e.g., file size) and "limits" to refer to vector quantities (e.g., the spatial boundary enveloping a dataset). Let's go with limits.